modules/nw-sriov-network-operator.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-network-operator_{context}"]
+
+= Configuring an SR-IOV Network Operator for virtual functions
+The Single Root I/O Virtualization (SR-IOV) network operator is used to allocate and configure Virtual Functions (VFs) from SR-IOV-supporting Physical Function NICs on the nodes.
```

Comment:
You could capitalise "network operator".

Reply:
Done.